.github/buildomat/jobs/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#:
+#: name = "build"
+#: variety = "basic"
+#: target = "helios"
+#: rust_toolchain = "nightly"
```

Comment:
I think we should keep this in sync with `rust-toolchain.toml`, so in this case perhaps we should:

```
#: rust_toolchain = "nightly-2021-11-24"
```